jbpm-system-test/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-parent</artifactId>
+    <!-- it must be a final one as the image must be on dockerhub -->
+    <version>7.57.0.Final</version>
```

Comment:
We can rely on snapshot here and have a separate property for a dockerhub image, but this can be changed in the future.

Reply:
As talked, I kept the unique kie-version because the "system-under-test" is the kie docker image, and it's using kie-ci and kie-server-client to run the scenarios; so, for avoiding incompatibilities I decided to use Final version also for these dependencies. Added an update-version script as you suggested